hw6_social.py

```diff
@@ -181,7 +301,14 @@ def graphStateCounts(stateCounts, title):
 Parameters: dict mapping strs to ints ; dict mapping strs to ints ; int ; str
 Returns: None
 '''
+import heapq
 def graphTopNStates(stateCounts, stateFeatureCounts, n, title):
+    featurerate={}
+    topstates={}
+    for i in stateFeatureCounts:
+        featurerate[i]=(stateFeatureCounts[i]/stateCounts[i])
+    topstates=dict(Counter(featurerate).most_common(n))
+    graphStateCounts(topstates, "Top n Featured")

```

Comment:
In graphTonNStates you should select **title** given in the function but instead you gave it as **Top n featured**

Reply:
corrected it